pystac_monty/sources/gdacs.py

```diff
@@ -382,6 +382,8 @@ def get_impact_category_from_sendai_c(
     ) -> MontyImpactExposureCategory:
         if sendainame == "houses damaged" or sendainame == "houses":
             return MontyImpactExposureCategory.BUILDINGS
+        elif sendainame == "houses destroyed" or sendainame == "houses":
+            return MontyImpactExposureCategory.BUILDINGS
```

Comment:
@Rup-Narayan-Rajbanshi  As both of them return the same thing `MontyImpactExposureCategory.BUILDINGS`, can we not use the elif and have all three in same if condition?